esx_service/utils/auth_api.py

```diff
@@ -190,7 +195,8 @@ def modify_privileges(privileges, allow_create, volume_maxsize_in_MB, volume_tot
     logging.debug("modify_privileges: allow_create=%s, volume_maxsize_in_MB=%s, volume_totalsize_in_MB=%s",
                   allow_create, volume_maxsize_in_MB, volume_totalsize_in_MB)
     if allow_create is not None:
```

Comment:
Ah... well, this is confusing to read. I'd recommend initializing allow_create so is never None